CHANGELOG.md

```diff
@@ -1,9 +1,19 @@
 # Changelog
 
+## [0.2.0] ｜ 2025-08-23
```

Comment:
Can you make it as: `## [0.2.0] ｜ Unrealeased` for now? 
I'll update it with the correct the release date later